src/fe/fe_interface.C

```diff
@@ -539,6 +539,11 @@ Point FEInterface::map(unsigned int dim,
                        const Elem * elem,
                        const Point & p)
 {
+#ifdef LIBMESH_ENABLE_INFINITE_ELEMENTS
+
+  if (is_InfFE_elem(elem->type()))
```

Comment:
I hadn't noticed this before, but this function's name does not match our usual naming scheme... that should probably be fixed, but not in this PR.